README.md

```diff
@@ -1,5 +1,7 @@
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](code_of_conduct.md)
 
+![ChemNLP logo](/assets/logo/logo_1.png)
```

Comment:
Hey Lemo, thanks for the PR. The logo is super big. Can you resize it?
See for example this guide: https://gist.github.com/uupaa/f77d2bcf4dc7a294d109